Makefile

```diff
@@ -0,0 +1,23 @@
+NAME = verypossible/serverless
+VERSION = 1.19.0
```

Comment:
From your next steps, is there a way to pull the latest version number of serverless from their site?

Reply:
It'd probably make sense to look at the releases via Github for that...it'd be slick, but just not sure what would be the event to trigger that.  Otherwise, maybe poll 1x day?

https://github.com/serverless/serverless/releases